lib/config/validation.ts

```diff
@@ -30,6 +30,7 @@ const options = getOptions();
 
 let optionTypes: Record<string, RenovateOptions['type']>;
 let optionParents: Record<string, RenovateOptions['parent']>;
+let optionGlobals: Set<string>;
```

Comment:
🤔 ` optionsGlobals` sounds strange. should we use `globalOptions` instead?

@rarkins @secustor WDYT?